src/routes/safe/components/Apps/components/ConfirmTxModal/ReviewConfirm.tsx

```diff
@@ -162,7 +162,7 @@ export const ReviewConfirm = ({
           origin: app.id,
           navigateToTransactionsTab: false,
           txNonce: txParameters.safeNonce,
-          safeTxGas: txParameters.safeTxGas ? txParameters.safeTxGas : undefined,
+          safeTxGas: txParameters.safeTxGas,
```

Comment:
Previously we were casting to Number here. This value can only be `string` or `undefined` so no need for ternary check here, if value is already undefined we can safely return the value